### PR TITLE
fix: Updated the docs for scorecard feature in IDP

### PR DESCRIPTION
**Pull request by Debanitrkl**
# Harness Developer Pull Request
The docs are updated according to the upcoming release of IDP. 
## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/internal-developer-portal/features/scorecard.md [hunk 1]
```diff
@@ -30,44 +30,32 @@ Scorecards play a pivotal role in ensuring software components are developed and
...
-  /harness/scorecard:
-    target: https://app.harness.io/
-    pathRewrite:
-      /api/proxy/harness/scorecard/?: /
-    allowedHeaders:
-    - authorization
-    - Harness-Account
-```
+- **Check**: A check is a query performed against a data point for a software component which results in either `Pass` or `Fail`.
+- **Data Source**: Data Sources are third-party providers which can provide a specific type of data for a software component. Data sources are fixed in IDP and have a specific way of enablement.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/features/scorecard.md:34]:
```suggestion
- **Data Source**: Data Sources are third-party providers which can provide a specific type of data for a software component. Example - GitHub, GitLab, Harness, Pagerduty, etc.
```

File: docs/internal-developer-portal/features/scorecard.md [hunk 2]
```diff
@@ -30,44 +30,32 @@ Scorecards play a pivotal role in ensuring software components are developed and
...
-    target: https://app.harness.io/
-    pathRewrite:
-      /api/proxy/harness/scorecard/?: /
-    allowedHeaders:
-    - authorization
-    - Harness-Account
-```
+- **Check**: A check is a query performed against a data point for a software component which results in either `Pass` or `Fail`.
+- **Data Source**: Data Sources are third-party providers which can provide a specific type of data for a software component. Data sources are fixed in IDP and have a specific way of enablement. 
+- **Data Points**: For each software componenet, every data source provides some data points. The data points could be a number, a string or a boolean.
```
Comment (jenil-harness) [inline docs/internal-developer-portal/features/scorecard.md:35]:
```suggestion
- **Data Points**: For each software component, every data source provides some data points. The data points could be a number, a string or a boolean. 
```

File: docs/internal-developer-portal/features/scorecard.md (hunk 1 above)
Comment (OrkoHunter) [reply docs/internal-developer-portal/features/scorecard.md:34]: Not sure about the "fixed" part. We'll be adding new data sources and in future, custom data source as well.

File: docs/internal-developer-portal/features/scorecard.md [hunk 3]
```diff
@@ -30,44 +30,32 @@ Scorecards play a pivotal role in ensuring software components are developed and
...
+- **Check**: A check is a query performed against a data point for a software component which results in either `Pass` or `Fail`.
+- **Data Source**: Data Sources are third-party providers which can provide a specific type of data for a software component. Data sources are fixed in IDP and have a specific way of enablement. 
+- **Data Points**: For each software componenet, every data source provides some data points. The data points could be a number, a string or a boolean. 
 
-<docimage path={require('../../internal-developer-portal/features/static/admin-ci-cd.png')}/>
+</details>
 
-<docimage path={require('../../internal-developer-portal/features/static/proxy.png')}/>
 
+#### Enable Scorecards
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/features/scorecard.md:40]: Can we have a "disable scorecards" as well?

Plus I feel "How to enable Scorecards in IDP" is probably more descriptive and catches the attention.

File: docs/internal-developer-portal/features/scorecard.md [hunk 4]
```diff
@@ -30,44 +30,32 @@ Scorecards play a pivotal role in ensuring software components are developed and
...
+- **Check**: A check is a query performed against a data point for a software component which results in either `Pass` or `Fail`.
+- **Data Source**: Data Sources are third-party providers which can provide a specific type of data for a software component. Data sources are fixed in IDP and have a specific way of enablement. 
+- **Data Points**: For each software component, every data source provides some data points. The data points could be a number, a string or a boolean. 
 
-<docimage path={require('../../internal-developer-portal/features/static/admin-ci-cd.png')}/>
+</details>
 
-<docimage path={require('../../internal-developer-portal/features/static/proxy.png')}/>
 
+#### Enable Scorecards
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/features/scorecard.md:40]:
```suggestion
## Enable Scorecards
```

Isn't this a larger section of this page given the section below with 3 # seems like a child of the enable scorecards section?

File: docs/internal-developer-portal/features/scorecard.md [hunk 5]
```diff
@@ -30,44 +30,32 @@ Scorecards play a pivotal role in ensuring software components are developed and
...
 ### Scorecard Components in IDP
 
 Scorecard has two main UI components which are developer facing and lives in the Catalog - 1. A small `Card` for the Overview page with the scores and, 2. A `Tab` view with details of the checks and how the score got computed. This is illustrated below. The Tab view contains detailed comprehensive information as shown in the image under [overview](/docs/internal-developer-portal/features/scorecard#overview)
 
 <docimage path={require('../../internal-developer-portal/features/static/scorecard-overviewpage.png')}/>
 
-2. #### Adding Card and Tab Content for an Entity
+:::info
+
+For the steps **1** and **2** the `Layout` is autopopulated with the adequate fields in the recent version of IDP and could be skipped, but users are still advised to check for the same on their `Layout` for the software components.
```
Comment (OrkoHunter) [inline docs/internal-developer-portal/features/scorecard.md:50]:
```suggestion
For the steps **1** and **2** the `Layout` is prepopulated with the adequate fields in the recent version of IDP and could be skipped, but users are still advised to check for the same on their `Layout` for the software components.  
```

File: docs/internal-developer-portal/features/scorecard.md (hunk 3 above)
Comment (OrkoHunter) [reply docs/internal-developer-portal/features/scorecard.md]: Makes sense - thank you Deba!

**Pull request merged**